routes.go

```diff
@@ -207,6 +209,12 @@ func (s *Server) handleEvent() http.HandlerFunc {
 				formatJSONError(w, http.StatusInternalServerError, err.Error())
 				return
 			}
+			// Marshal event for production to kafka topic
+			message, err := json.Marshal(&e)
```

Comment:
`message` is a Kafka-specific vocabulary word. If the goal of using a channel here is to abstract away the fact that the channel is being read by a Kafka producer, then we should avoid the use of "message" here. Sometimes I use nested scopes to reuse a variable name to increase clarity. Something like:

```go
{
    data, err := json.Marshal(&e)
    if err != nil {
        log.Errorf("error: failed to Marshal event into JSON: %v", err)
    }
    s.events <- data
}
```

Reply:
I had no idea you could nest scopes this way 0_0

TIL